pymarc/record.py

```diff
@@ -444,6 +444,27 @@ def isbn(self):
             pass
         return None
 
+
+    def lcc_controlnumber(self):
+        """
+        Returns the Library of Congress Classification (LCC)
+        Control Number of the record. 
+        """
+        
+        if self['010']:
+            return self['010'].format_field()
+        return None
+        
+    def lcc_callnumber(self):
+        """
+        Returns the Library of Congress Classification (LCC)
+        Call Number of the record.
+        """
+        
+        if self['050']:
```

Comment:
The second indicator in 050 is "0" if the call number was assigned by the LoC, "4" if assigned by some other entity.